docs/appkit/next/solana/about/programs.mdx

```diff
@@ -23,7 +23,7 @@ function deserializeCounterAccount(data?: Buffer): { count: number } {
   }
 }
 
-const { address, currentChain } = useAppKitAccount()
+const { address } = useAppKitAccount()
```

Comment:
Have you tested this out?

Reply:
https://github.com/reown-com/web-examples/pull/729